IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -231,4 +231,11 @@ - (NSUInteger)refreshInterval
     return kDefaultPRTRefreshInterval;
 }
 
+- (NSString *)prtId
+{
+    NSString *accountId = self.accountIdentifier.homeAccountId ?: @"";
+    NSString *deviceId = self.deviceID ?: @"";
+    return [NSString stringWithFormat:@"%@%@", accountId, deviceId];
```

Comment:
Just double check with Vishakha that there's no lenght limit for prtId

Reply:
As discussed Updated to hash of accountId and deviceId, As ESTS has limit of 10.